deploy/api.yaml

```diff
@@ -85,7 +126,52 @@ components:
       properties:
         session_id:
           type: "string"
-x-amazon-apigateway-request-validators:
-  Validate body:
-    validateRequestParameters: false
-    validateRequestBody: true
+    PostcodeLookupResponse:
```

Comment:
Could you put the `x-amazon-apigateway-request-validators:` back in please to avoid a regression on the /session API? You could add a separate validator here for the postcode GET that uses `validateRequestParameters: true`. This should allow API gateway to reject missing {postcode} in the path